docs/contract-testing-against-schemas.md

```diff
@@ -1,46 +1,49 @@
 # Contract Testing against Schemas
 
-We want to be able to evolve our publishing systems and have confidence that the changes we make will not break something.
+When changes are made to govuk-content-schemas, the test suite of each of the [dependent apps](https://github.com/alphagov/govuk-content-schemas/blob/aad248a867e878a496dcb488a4ba2170cebdb86b/Jenkinsfile#L7-L38) is run against that branch, to ensure that any changes are compatible.
 
-We could do this by running all the systems at the same time and testing end-to-end. However setting up such a configuration would be tricky, and keeping it working reliably in a reproducible way which avoids test interference would be very difficult.
+This method of contract testing is different to our usual [Pact tests](https://docs.publishing.service.gov.uk/manual/pact-broker.html).
 
-The idea of contract testing is to gain more confidence that the whole system works together properly, but without needing to run all the services at the same time. Instead we can test each service in isolation using a 'contract' which describes the expectations on each interacting service.
+## Background
+
+govuk-content-schemas contract testing was proposed in [RFC 4](https://gov-uk.atlassian.net/wiki/display/WH/RFC+4+%3A+Enabling+the+independent+iteration+of+formats+on+government-frontend) and described in the GOV.UK blog post "[Validating a distributed architecture with JSON Schema](https://gdstechnology.blog.gov.uk/2015/01/07/validating-a-distributed-architecture-with-json-schema/)".
 
-The definition of the contracts are provided by [govuk-content-schemas](https://github.com/alphagov/govuk-content-schemas).
+We wanted to be able to evolve our publishing systems and have confidence that the changes we make will not break something. We could do this by running all the systems at the same time and testing end-to-end, but setting up such a configuration would be tricky, and keeping it working reliably in a reproducible way which avoids test interference would be very difficult.
 
-For more information and background please read the [README.md](../README.md)
+The idea of contract testing is to gain more confidence that the whole system works together properly, but without needing to run all the services at the same time. Instead we can test each service in isolation using a 'contract' which describes the expectations on each interacting service.
 
-## Overview of publishing
+## How govuk-content-schemas contract testing works
```

Comment:
I think this is actually about how the different schemas are generated, rather than the contract testing itself. Could we change this to clarify that?

You could move the last two paragraphs to the next section, as they're more about the principle of the tests.